04-14 14:16:00 imba: False
04-14 14:16:00 model_name: DANN
04-14 14:16:00 Domain: SEU
04-14 14:16:00 source: SEU_1
04-14 14:16:00 target: SEU_0
04-14 14:16:00 data_dir: ./datasets
04-14 14:16:00 train_mode: single_source
04-14 14:16:00 cuda_device: 0
04-14 14:16:00 save_dir: ./ckpt/DANN/single_source
04-14 14:16:00 max_epoch: 100
04-14 14:16:00 batch_size: 64
04-14 14:16:00 num_workers: 0
04-14 14:16:00 signal_size: 2048
04-14 14:16:00 random_state: 75
04-14 14:16:00 normlizetype: mean-std
04-14 14:16:00 opt: sgd
04-14 14:16:00 lr: 0.001
04-14 14:16:00 momentum: 0.9
04-14 14:16:00 betas: (0.9, 0.999)
04-14 14:16:00 weight_decay: 0.0005
04-14 14:16:00 lr_scheduler: stepLR
04-14 14:16:00 gamma: 0.2
04-14 14:16:00 steps: 10
04-14 14:16:00 tradeoff: ['exp', 'exp', 'exp']
04-14 14:16:00 dropout: 0.0
04-14 14:16:00 save: False
04-14 14:16:00 load_path: 
04-14 14:16:00 tsne: False
04-14 14:16:00 save_path: ./ckpt/DANN/single_source/[SEU_1]To[SEU_0]_0414-141600
04-14 14:16:00 Detect 10 classes: ['1normal_bearing', '1normal_gear', 'ball', 'chipped', 'comb', 'inner', 'miss', 'outer', 'root', 'surface']
04-14 14:16:01 using 1 / 1 gpus
Loading source: SEU, Condition: 1
Traceback (most recent call last):
  File "train.py", line 89, in <module>
    trainer = importlib.import_module(f"models.{args.model_name}").Trainset(args)
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/models/DANN.py", line 36, in __init__
    self._init_data()
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/train_utils.py", line 103, in _init_data
    self.datasets[source] = Dataset(data_root, src, args.faults, args.signal_size, args.normlizetype, condition=condition
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/data_loader/source_load.py", line 58, in __init__
    self.train_data, self.train_labels = get_files(root=data_dir, dataset=dataset, faults=faults, signal_size=signal_size, condition=condition)
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/data_loader/source_load.py", line 17, in get_files
    for item in os.listdir(data_dir):
FileNotFoundError: [Errno 2] No such file or directory: './datasets/SEU/condition_1/1normal_gear'
